contrib/python-bindings/include/point_wrapper.h

```diff
@@ -51,12 +56,89 @@ namespace python
     ~PointWrapper();
 
     /**
+     * Return the Euclidean distance of this point to the point p, i.e., the
+     * l2_norm of the difference between the vectors representing the two
+     * points.
+     */
+    double distance(const PointWrapper &p) const;
+
+    /**
+     * Return the l2_norm of the vector.
```

Comment:
maybe rephrase along the lines that this is a norm of the vector connecting origin to the given point.